packages/uxpin-merge-cli/src/steps/serialization/component/implementation/typescript/component/isFunctionalComponentWithReactForwardRef.ts

```diff
@@ -0,0 +1,40 @@
+import * as ts from 'typescript';
+
+function extractExpression(node:ts.Node):ts.Identifier|ts.PropertyAccessExpression|undefined {
+  if (ts.isCallExpression(node)) {
+    return node.expression as ts.Identifier;
+  }
+
+  if (ts.isExportAssignment(node)) {
+    return (ts.isCallExpression(node.expression) ? node.expression.expression : node.expression) as ts.Identifier;
+  }
+
+  if (ts.isVariableStatement(node)
+		&& ts.isCallExpression(node.declarationList?.declarations[0]?.initializer as ts.Node)
+	) {
+    return extractExpression(node.declarationList.declarations[0].initializer as ts.CallExpression);
+  }
+
+  if (ts.isVariableDeclaration(node) && ts.isCallExpression(node.initializer as ts.CallExpression)) {
+    return extractExpression(node.initializer as ts.CallExpression);
+  }
```

Comment:
Can you add comments describing the syntax for each case like you did in findExportedFunctionWithReactForwardRef.ts? 🙏